pyproject.toml

```
@@ -0,0 +1,65 @@
+[build-system]
+requires = ["setuptools", "wheel", "versioneer"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "MembraneCurvature"
+version = "1.0.0"
```

Because versioneer (or alternative) is handling the versioning, this should be put under the `dynamic` entry instead.

```suggestion
dynamic = ["version"]
```